pontoon/base/migrations/0088_add_fulltext_search.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-03 02:23
+from __future__ import unicode_literals
+
+import django.contrib.postgres.search
+from django.contrib.postgres.operations import TrigramExtension, CreateExtension
+from django.db import migrations
+
+from pontoon.db.migrations import (
+    MultiFieldTRGMIndex,
+)
+
+entity_document_update_sql = """
+    UPDATE base_translation AS t 
+      SET entity_document = (e.key || ' ' || e.string || ' ' || e.string_plural || ' ' || e.comment)
+      FROM base_entity AS e
+      WHERE t.entity_id=e.id;
+"""
```

Comment:
1. I just assumed that raw sql query will be faster than anything.
2. I can try to implement your suggestion and check what's faster.